solutions/day-11/level2/src/App.js

```diff
@@ -0,0 +1,35 @@
+import { useRef, useState } from "react";
+import FloatingView from "./components/FloatingView";
+import "./styles/App.css";
+
+const App = () => {
+  const [coordinate, setCoordinate] = useState({ x: 0, y: 0 });
+  const containerRef = useRef();
+  const floatingViewRef = useRef();
+
+  const getRandomInt = (max, min = 0) => {
+    min = Math.ceil(min);
+    max = Math.floor(max);
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+  };
+  const handleMouseEnter = () => {
+    const container = containerRef.current;
+    const floatingView = floatingViewRef.current;
+
+    const x = getRandomInt(container.clientWidth - floatingView.clientWidth);
+    const y = getRandomInt(container.clientHeight - floatingView.clientHeight);
+    setCoordinate({ x: x, y: y });
```

Comment:
In ES6, you can apply the object literal property shorthand. It enables you to pass in the name of the key as opposed to repeating the name and the key.
```
setCoordinate({ x, y });
```
Similar to https://github.com/jhrasco-ct/30-Days-Of-React/pull/8#discussion_r1037100856